source/common/router/router.cc

```diff
@@ -1333,16 +1333,22 @@ Filter::UpstreamRequest::UpstreamRequest(Filter& parent, Http::ConnectionPool::I
         parent_.callbacks_->tracingConfig(), "router " + parent.cluster_->name() + " egress",
         parent.timeSource().systemTime());
     span_->setTag(Tracing::Tags::get().Component, Tracing::Tags::get().Proxy);
```

Comment:
This tag is common to all spans, so should be moved to `setCommonTags`.

Reply:
Done